requirements.txt

```diff
@@ -3,11 +3,15 @@ Flask-Cors==5.0.0
 python-dotenv==1.0.1
 sentry-sdk==2.13.0
 marshmallow==3.22.0
-pytest==8.3.2
```

Comment:
This does not make sense to be added to the `requirements.txt` file as it should be only used for testing.